api/src/org/apache/cloudstack/api/command/admin/network/CreateNetworkCmdByAdmin.java

```diff
@@ -48,6 +51,13 @@ public String getVlan() {
         return vlan;
     }
 
+    public Boolean getBypassVlanOverlapCheck() {
```

Comment:
Is there a reason why we do return a Boolean instead of boolean? (this function can never return null)

Reply:
Yes, because cmd classes set API args to fields using reflections etc. It's easier to not use native type `boolean`. When arg is not sent part of the API request, it would be set to null; setting null to `boolean` will throw an exception.